### PR TITLE
tensorflow: CVE-2024-7592

### PR DESCRIPTION
**Pull request by bfjelds**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./LICENSES-AND-NOTICES/SPECS/data/licenses.json`, `./LICENSES-AND-NOTICES/SPECS/LICENSES-MAP.md`, `./LICENSES-AND-NOTICES/SPECS/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 
Take python CVE-2024-7592 patch and modify to apply it to SOURCE1

###### Change Log  
- Take python patch and modify to apply it to SOURCE1

###### Does this affect the toolchain?  
**NO**

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2024-7592

###### Test Methodology
- https://dev.azure.com/mariner-org/mariner/_build/results?buildId=629308&view=results

### CODE REVIEW

File: SPECS/tensorflow/tensorflow.spec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Summary:        TensorFlow is an open source machine learning framework for everyone.
 Name:           tensorflow
 Version:        2.16.1
-Release:        5%{?dist}
+Release:        6%{?dist}
```
Comment (mfrw) [inline SPECS/tensorflow/tensorflow.spec:4]: Double Checking:
The CVE seems to be in the vendored: `python_aarch64-unknown-linux-gnu/lib/python3.12/http/cookies.py`

Comment (bfjelds) [reply SPECS/tensorflow/tensorflow.spec:4]: i thought the CVE would be addressed by `BuildRequires: python3-devel` picking up the fixes in python3 package?

Comment (mfrw) [reply SPECS/tensorflow/tensorflow.spec:4]: Honestly I am not sure, so I just thought of double checking :)
Would be a good `note-2-self` :)

Comment (bfjelds) [reply SPECS/tensorflow/tensorflow.spec:4]: added the python patch and applied it to SOURCE1

**Pull request merged**